code/modules/reagents/Chemistry-Goon-Medicine.dm

```diff
@@ -326,6 +325,7 @@ datum/reagent/ephedrine/on_mob_life(var/mob/living/M as mob)
 	M.AdjustStunned(-1)
 	M.AdjustWeakened(-1)
 	M.adjustStaminaLoss(-1*REM)
+	M.adjustOxyLoss(-1*REM)
```

Comment:
Ephedrine shouldn't be impacting O2 damage or losebreath; likewise, it shouldn't really speed you up either; leave that for Methamphetamine.